gwpy/cli/cliproduct.py

```diff
@@ -729,7 +729,8 @@ def add_segs(self, args):
             start = None
             end = 0
             for ts in self.timeseries:
-                if start:
+                if start is not None:
+                    # NB: quantity truth value is deprecated
```

Comment:
```suggestion
```
I suggest removing this note, it's a simple enough change that doesn't touch any functionality that this isn't needed